diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0c7d9c68d..83e1ae4b0 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -264,7 +264,7 @@ if (NOT cxx_std_11 IN_LIST CMAKE_CXX_COMPILE_FEATURES)
 endif()
 # Boost
 if(NOT DEFINED Boost_INCLUDE_DIR OR NOT DEFINED Boost_LIBRARIES)
-	find_package(Boost REQUIRED COMPONENTS ${required_boost_components})
+	find_package(Boost REQUIRED COMPONENTS date_time ${required_boost_components})
 endif()
 # now test the second requirement:
 if (Boost_VERSION VERSION_GREATER_EQUAL 106600)
